### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Delawen**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+There are many ways to contribute to Kaoto:
+
+## Issues
+
+### Submit a new issue
+
+If you have a bug or a suggestion for a new feature, you can create new issues.
+Always **use one of the templates available** and answer as many of the questions as you can.
+
+If you are **submitting a bug**, provide a simple step by step explanation on how to reproduce it and what is the expected outcome.
```
Comment (kahboom) [inline CONTRIBUTING.md:14]: >step by step explanation _of_ how to reproduce

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+If you are **submitting a feature**, be ready to follow up and contribute with its development. Features that are proposed but don't have
+funds or developers ready to implement it may be closed due to not enough interest raised. If you can't fund or implement it yourself and 
+you want the feature implemented, you must look for a way to find resources to implement it.
```
Comment (kahboom) [inline CONTRIBUTING.md:18]: what if it's a feature we like the idea of? 😆

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+If you are reviewing pull requests, please use the [conventional comments](https://conventionalcomments.org/) standard to do so. 
+Comments that don't follow this standard may be ignored.
+
+There are a few things to consider when sending a pull request merge:
+
+ * Small commits. We prefer small commits because they are easier to review
+ * All commits must pass tests: Each commit should have consistency on its own and don't break any functionality
+ * All jobs/checks must be green: This includes test coverage, code smells, security issues,... 
+ * Be descriptive on the PR text about what the changes are. Better to have duplicated explanation than no explanation at all. Provide examples.
+ * Add screenshots and videos of what your PR is doing. Specially if you are adding a new feature.
```
Comment (kahboom) [inline CONTRIBUTING.md:44]: > of what your PR is doing, _especially_ if you are adding a new feature.

**Review by kahboom — CHANGES_REQUESTED** [review]
minor nitpicks

File: CONTRIBUTING.md (hunk 2 above)
Comment (Delawen) [reply CONTRIBUTING.md:18]: Then we are the resource to implement it. But better if someone else does.

**Review by kahboom — APPROVED** [review]

**Pull request merged**